### PR TITLE
More istypes replaced with helpers

### PR DESCRIPTION
**Pull request by ChangelingRain**
...

### CODE REVIEW

File: code/game/machinery/computer/message.dm [hunk 1]
```diff
@@ -247,7 +247,7 @@
 	if(..())
 		return
 
-	if ((usr.contents.Find(src) || (in_range(src, usr) && istype(src.loc, /turf))) || (istype(usr, /mob/living/silicon)))
+	if(usr.contents.Find(src) || (in_range(src, usr) && isturf(loc)) || istype(usr, /mob/living/silicon))
```
Comment (CyberbossJHCB) [inline code/game/machinery/computer/message.dm]: Missed an issilicon

File: code/modules/awaymissions/bluespaceartillery.dm [hunk 2]
```diff
@@ -39,11 +39,13 @@
 	if(..())
 		return
 	var/A
-	A = input("Area to jump bombard", "Open Fire", A) in teleportlocs
+	A = input("Area to  bombard", "Open Fire", A) in teleportlocs
```
Comment (CyberbossJHCB) [inline code/modules/awaymissions/bluespaceartillery.dm]: Double space

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Sorry for being a pedantic little shit

**Review by Cyberboss — APPROVED** [review]
I'm getting serious deja vu from approving this

**Pull request merged**